invoices/invoiceregistry.go

```diff
@@ -503,11 +502,8 @@ func (i *InvoiceRegistry) AddInvoice(invoice *channeldb.Invoice,
 
 	i.Lock()
 
-	log.Debugf("Invoice(%v): added %v", paymentHash,
-		newLogClosure(func() string {
-			return spew.Sdump(invoice)
-		}),
-	)
+	log.Debugf("Invoice(%v): added with terms %v", paymentHash,
+		invoice.Terms)
```

Comment:
What about the other invoice fields? Not sure in what way this statement has been useful historically for debugging.

Reply:
the remaining fields from `ContractTerms` are payment addr and features, we could log payment addr pretty easily but logging the features in a concise way is a little more work. i left them off bc idt they add much here, but like you idk what the original intention was. i assumed it was just a catch all